checkers.py

```diff
@@ -0,0 +1,63 @@
+def people(text):
+    if not "landlord" in text or not "tenant" in text:
+        return "The landlord or tenant field might be missing"
+
+    return "Included"
+
+def starting_date(text):
+    synonyms = ["starting date","move in date", "starts on:"]
+    for synonym in synonyms:
+        if synonym in text:
+            return "Included"
+    return "The starting date might be missing."
+
+def address(text):
+    return "Included"
+
+def end_date(text):
+    synonyms = ["continues on a month-to-month basis", "continues on another periodic basis", "a fixed term ending on"]
```

Comment:
we could potentially have these in json files so that we can have bunch of if statements